jest.config.js

```diff
@@ -8,12 +8,14 @@ module.exports = {
     // Jest doesn't care about styles, images, fonts, etc
     '\\.(css|scss|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
     '<rootDir>/tools/tests/unit/__mocks__/fileMock.js',
-    // Webpack aliases. @TODO: pull in webpack.shared.config.js and pull this piece off?
+
+    // Webpack aliases. This is a pain, but Babel does NOT understand Webpack resolved aliases
+    // @TODO: Look into helpers for this (e.g. https://github.com/mwolson/jest-webpack-alias)
     '^protons$': '<rootDir>/source/_patterns/00-protons/',
-    '^atoms$': '<rootDir>/source/_patterns/01-atoms/',
-    '^molecules$': '<rootDir>/source/_patterns/02-molecules/',
-    '^organisms$': '<rootDir>/source/_patterns/03-organisms/',
-    '^templates$': '<rootDir>/source/_patterns/04-templates/',
-    '^pages$': '<rootDir>/source/_patterns/05-pages/',
+    '^atoms[/](.+)': '<rootDir>/source/_patterns/01-atoms/$1',
```

Comment:
Why define a character class for a single character?

Reply:
My regex is not great. Are you asking about the `[/]`?

Reply:
Yes, I don't understand why it wouldn't just be this:

```
'^atoms/(.+)'
```

Reply:
Note forward slash is a default regex delimiter in a lot of places, so that may have to be `^atoms\/(.+)` if babel gets complainy.

Reply:
It works! Pushing updates now.